src/test/test_integration_queues.py

```diff
@@ -0,0 +1,13 @@
+#This is the integration test file which i'm working on it now
+
+import sys
+sys.path.insert(1, 'D:/Logabit/pipeforce-service-template-python-1/src/service')
+sys.path.append("../service/")
+import main
+
+
+def test_statuscheck():
```

Comment:
Please create a separate method for each test test_status, test_exists, ...

Reply:
Ok, I'll create separate method for each